tests/php/BackgroundJob/CheckHostedSignalingServerTest.php

```diff
@@ -81,15 +81,15 @@ public function getBackgroundJob(): CheckHostedSignalingServer {
 	public function testRunWithNoChange() {
 		$backgroundJob = $this->getBackgroundJob();
 
-		$this->config->expects($this->at(0))
+		$this->config
 			->method('getAppValue')
-			->with('spreed', 'hosted-signaling-server-account-id', '')
-			->willReturn('my-account-id');
-		$this->config->expects($this->at(1))
-			->method('getAppValue')
-			->with('spreed', 'hosted-signaling-server-account', '{}')
-			->willReturn('{"status": "pending"}');
-		$this->config->expects($this->at(2))
+			->willReturnCallback(function ($appid, $key, $default) {
```

Comment:
You can use `willReturnMap` instead (available on all versions too):

https://phpunit.readthedocs.io/en/9.5/test-doubles.html#test-doubles-stubs-examples-stubtest5-php